### PR TITLE
Chapter 19 terminal

### PR DESCRIPTION
**Pull request by jwoolbright23**

### CODE REVIEW

File: content/appendices/terminal-commands/pictures/locate_launchcode_courses.png.1
Comment (icr8FreeCode) [inline content/appendices/terminal-commands/pictures/locate_launchcode_courses.png.1:1]: not sure what this file is ?

File: content/appendices/terminal-commands/pictures/lc101_current_dir.png.3
Comment (icr8FreeCode) [inline content/appendices/terminal-commands/pictures/lc101_current_dir.png.3:1]: not sure what this file is ?

File: content/appendices/terminal-commands/pictures/lc101_current_dir.png.2
Comment (icr8FreeCode) [inline content/appendices/terminal-commands/pictures/lc101_current_dir.png.2:1]: not sure what this file is ?

File: content/appendices/terminal-commands/pictures/lc101_current_dir.png.1
Comment (icr8FreeCode) [inline content/appendices/terminal-commands/pictures/lc101_current_dir.png.1:1]: not sure what this file is ?

File: content/appendices/terminal-commands/pictures/init_tree.png.1
Comment (icr8FreeCode) [inline content/appendices/terminal-commands/pictures/init_tree.png.1:1]: not sure what this file is ?

File: content/appendices/terminal-commands/pictures/unit1_current_dir.png.1
Comment (icr8FreeCode) [inline content/appendices/terminal-commands/pictures/unit1_current_dir.png.1:1]: not sure what this file is ?

File: content/appendices/terminal-commands/pictures/unit1_current_dir.png.2
Comment (icr8FreeCode) [inline content/appendices/terminal-commands/pictures/unit1_current_dir.png.2:1]: not sure what this file is ?

**Review by icr8FreeCode — CHANGES_REQUESTED** [review]

File: content/appendices/terminal-commands/pictures/locate_launchcode_courses.png.1
Comment (jwoolbright23) [reply content/appendices/terminal-commands/pictures/locate_launchcode_courses.png.1:1]: Added directory structure within javascript-projects repo: https://github.com/LaunchCodeEducation/javascript-projects/tree/main/terminal-commands/launchcode_courses

This applies to this comment and all following.

**Review by icr8FreeCode — APPROVED** [review]

**Pull request merged**